tests/unit/allowable-actions-mixin.spec.ts

```diff
@@ -105,9 +105,11 @@ describe('Allowable Actions Mixin', () => {
     expect(vm.isAllowed(AllowableActions.BUSINESS_INFORMATION)).toBe(false)
 
     // verify both feature flag and allowed filing type
-    setFeatureFlag(true)
+    setFeatureFlag('special-resolution-ui-enabled')
     setAllowedFilingType({ name: FilingTypes.SPECIAL_RESOLUTION })
-    expect(vm.isAllowed(AllowableActions.BUSINESS_INFORMATION)).toBe(true)
```

Comment:
This test was returning false for some reason. I've spent some time trying to fix it. Ditto lines 155, 165, 184, and 369.

Reply:
Yes, makes sense. I was struggling with finding a fix for this test. I'll create a ticket and mention there all the tests that I'll be disabling here.

Reply:
New ticket: https://github.com/bcgov/entity/issues/16525

Reply:
Reverted, thanks.